modules/transactions/src/main/java/org/apache/ignite/internal/tx/message/RwTransactionsFinishedResponse.java

```diff
@@ -20,7 +20,10 @@
 import org.apache.ignite.network.NetworkMessage;
 import org.apache.ignite.network.annotations.Transferable;
 
-/** Request to check that the requested catalog version has been reached and all RW transactions prior to it have completed. */
+/**
+ * Response to check that the requested catalog version has been reached and all RW transactions started on versions before it have
```

Comment:
```suggestion
 * Response to check that the requested catalog version has been activated and all RW transactions started on versions before it have
```

It seems we need to be specific that is activation which is reached.

Reply:
fix it